arm-servicefabric/2016-09-01/swagger/servicefabric.json

```diff
@@ -458,55 +458,59 @@
           ]
         },
         "applicationPorts": {
-          "description": "Application ports",
+          "description": "Ports used by applications",
           "$ref": "#/definitions/EndpointRangeDescription"
         },
         "ephemeralPorts": {
-          "description": "Ephemeral ports",
+          "description": "System assgined application ports",
           "$ref": "#/definitions/EndpointRangeDescription"
         },
         "isPrimary": {
-          "description": "Primary node or not",
+          "description": "Mark this as the primary node type",
           "type": "boolean"
         },
         "vmInstanceCount": {
-          "description": "Instance count",
+          "description": "The number of node instances in the node type",
           "type": "integer"
         },
         "reverseProxyEndpointPort": {
-          "description": "Reverse proxy endpoint port",
+          "description": "Endpoint used by reverse proxy",
           "type": "integer"
         }
       }
     },
     "ClientCertificateThumbprint": {
-      "description": "NodeType description",
+      "description": "Client certificate details using thumbprint",
       "properties": {
         "isAdmin": {
           "type": "boolean",
-          "description": "Is admin or not"
+          "description": "Is this certificate used for admin access from the client, if false , it is used or query only access"
         },
         "certificateThumbprint": {
-          "description": "ClientCertificate thumbprint",
+          "description": "Certificate thumbprint",
           "type": "string"
         }
       }
     },
     "ClientCertificateCommonName": {
-      "description": "NodeType description",
+      "description": "Client certificate details using common name",
       "properties": {
         "isAdmin": {
           "type": "boolean",
-          "description": "Is the certificate for admin usage or not"
+          "description": "Is this certificate used for admin access from the client, if false , it is used or query only access"
+        },
+        "certificateCommonName": {
+          "description": "Certificate common name to be granted access; be carefull using wild card common names",
+          "type": "string"
         },
         "certificateIssuerThumbprint": {
           "description": "Certificate issuer thumbprint",
           "type": "string"
-        }
+        }        
       }
     },
     "ClusterEnvironment": {
-      "description": "Cluster environment",
+      "description": "Cluster operating system",
```

Comment:
In description, please explain what is meant by Default clusterOperatingSystem. Will that be windows or linux?